utility/utils.py

```diff
@@ -602,33 +615,37 @@ def email_results(results_list, run_id, trigger_user, run_dir, suite_run_time, s
             log.exception(e)
 
 
-def create_html_file(run_name, results_list, run_time, user, run_dir) -> str:
+def create_html_file(run_name, results_list, run_time, user, run_dir, info) -> str:
     """
     Creates the HTML file from the template to be sent via mail
     Args:
         results_list (list): test case results info
         run_name (str): name of the test run
         user (str): user of the node where the run is triggered from
         run_dir (str): log directory path
-        run_time (str): suite total duration info
+        run_time (dict): suite total duration info
+        info (dict): General information about the test run
     Returns: HTML file
 
     """
     log_link = "http://magna002.ceph.redhat.com/cephci-jenkins/{run}/".format(run=run_name)
+    info["link"] = f"{log_link}/startup.log"
```

Comment:
Let's keep the hyperlinks `relative`. It facilities... moving to different locations in case of local runs.